### ISSUE TITLE
Setup instructions incomplete

### ISSUE THREAD

**Issue by daniel-heywood** [armory3d/armory#1502]
Being new to all this (github, blender, armory3D), my apologies if this is not the place to note this.

Regarding: 
Setup instructions found here: https://github.com/armory3d/armory/wiki/setup

Missing the final line of instruction:

—Inside of Blender Preferences: Add-ons, under Render: Armory, fill the "SDK Path" field by clicking on the folder icon, then navigating to the location you have stored the Armory SDK folder, and then click "Accept"

Didn't know this until I watched a video by Lumpology on Youtube titled "How To Install Armory In Blender 2.8"

Also, being new to blender, I had not realized that this instruction:
 
"Enable Armory add-on in Blender"

... meant to do this:

"Click the checkbox next to Render: Armory from within Preferences: Add-ons."

**Comment by N8n5h** [comment]
> —Inside of Blender Preferences: Add-ons, under Render: Armory, fill the "SDK Path" field by clicking on the folder icon, then navigating to the location you have stored the Armory SDK folder, and then click "Accept"
Didn't know this until I watched a video by Lumpology on Youtube titled "How To Install Armory In Blender 2.8"

The reason for why that step is missing is because the `auto-detect SDK` feature of the armory.py should automatically fill the SDK directory for you when you enable the addon, when downloading Armory from https://armory3d.org/download.html.

Blender 2.81 removed a feature that the `auto-detect SDK` used. But later Armory acommodated and it was replaced so it should work again.

Also I updated the setup instructions and added your detail of the step, thanks. :+1:
